### ISSUE TITLE
Error editing initial content on pull request

### ISSUE THREAD

**Issue by ryanburnette** [go-gitea/gitea#4755]
- Gitea version (or commit ref): 1.4.0
- Git version: NA
- Operating system: Linux
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No
  - [ ] Not relevant
- Log gist:

## Description

I created a PR with no content. When trying to edit the initial comment I got this error in the console. I couldn't reproduce it on try.gitea.io because I couldn't get creating a PR to work in the first place there.

The error in console reads:

```
index.js?v=1bf6e69c18341244d990250bf5aa3ce0:573 Uncaught ReferenceError: issuesTribute is not defined
    at HTMLAnchorElement.<anonymous> (index.js?v=1bf6e69c18341244d990250bf5aa3ce0:573)
    at HTMLAnchorElement.dispatch (jquery.min.js:4)
    at HTMLAnchorElement.r.handle (jquery.min.js:4)
```

## Screenshots

![screen shot 2018-08-21 at 10 08 19 am](https://user-images.githubusercontent.com/2252601/44406653-28695000-a52a-11e8-9a39-faa0b0761a8d.png)

**Comment by nicoulaj** [comment]
For what it's worth, I could reproduce this issue on https://projects.torsion.org which uses gitea version 2d707fe by trying to edit an issue comment (https://projects.torsion.org/witten/borgmatic/issues/116#issuecomment-1017)

**Comment by lunny** [comment]
should also be fixed by #5516
